hipamd/hip-config-amd.cmake

```diff
@@ -133,7 +133,7 @@ if(HIP_CLANG_NUM_PARALLEL_JOBS GREATER 1)
     endif()
     hip_add_interface_link_flags(hip::device -parallel-jobs=${HIP_CLANG_NUM_PARALLEL_JOBS})
   else()
-    message("clang compiler doesn't support parallel jobs")
+    message(DEBUG "clang compiler doesn't support parallel jobs")
```

Comment:
I am not sure how often we hit this case in practice, but the message here is more useful from a user perspective than from a dev perspective I think as it flags a property of the host compiler (CMAKE_CXX_COMPILER that is in users control), I would leave this as it is.